text_generation/causal_lm/cpp/continuous_batching/python/tests/test_sampling.py

```diff
@@ -97,12 +98,22 @@ class RandomSamplingTestStruct:
     RandomSamplingTestStruct(generation_config=get_multinomial_temperature_and_repetition_penalty(),
                              prompts=["What is OpenVINO?"],
                              ref_texts=[ ["\nOpen Vino's are a new and improved way to find cheap, fast-investment frozen vegetables that have no waste or calories. They're"] ]),
+    RandomSamplingTestStruct(generation_config=get_multinomial_max_and_min_token(),
+                             prompts=["What is OpenVINO?"],
+                             ref_texts=[ ["\nOpenVINO is an open-source project that allows developers to create, use, and monetize content for the OpenVINO Platform. Open"] ]),
 ]
 
 
 @pytest.mark.precommit
 @pytest.mark.parametrize("test_struct", RANDOM_SAMPLING_TEST_CASES,
-        ids=["multinomial_temperature", "multinomial_temperature_and_top_p", "multinomial_temperature_and_top_k", "multinomial_temperature_top_p_and_top_k", "multinomial_temperature_and_repetition_penalty"])
+        ids=[
+                "multinomial_temperature",
+                "multinomial_temperature_and_top_p",
+                "multinomial_temperature_and_top_k",
+                "multinomial_temperature_top_p_and_top_k",
+                "multinomial_temperature_and_repetition_penalty",
+                "multinomial_max_and_min_token"
```

Comment:
can we add test for non-multinomial cases? E.g. greedy with penalties can also work with `min_tokens`

I wonder whether `min_tokens` is also applicable for beam search? Looks like yes, but we can handle beam search case separately, because it can be non-trivial to handle it.